ui/src/app/pages/analysis/leonardo-app-launcher.tsx

```diff
@@ -55,13 +56,6 @@ export enum LeoApplicationType {
   SparkConsole,
 }
 
-export enum SparkConsolePath {
```

Comment:
moved to runtime-utils, on the principle that **components** should not depend on **pages**